dissect/target/plugins/os/unix/applications.py

```diff
@@ -0,0 +1,69 @@
+from typing import Iterator
+
+from dissect.target.exceptions import UnsupportedPluginError
+from dissect.target.helpers import configutil
+from dissect.target.helpers.fsutil import TargetPath
+from dissect.target.helpers.record import UnixApplicationRecord
+from dissect.target.plugin import Plugin, export
+from dissect.target.target import Target
+
+
+class UnixApplicationsPlugin(Plugin):
+    """Unix Applications plugin."""
+
+    SYSTEM_PATHS = [
+        "/usr/share/applications/",
+        "/usr/local/share/applications/",
+        "/var/lib/snapd/desktop/applications/",
+        "/var/lib/flatpak/exports/share/applications/",
+    ]
+
+    USER_PATHS = [
+        ".local/share/applications/",
+    ]
+
+    SYSTEM_APPS = [
+        "org.gnome.",
+    ]
+
+    def __init__(self, target: Target):
+        super().__init__(target)
+        self.desktop_files = list(self.find_desktop_files())
+
+    def find_desktop_files(self) -> Iterator[TargetPath]:
```

Comment:
```suggestion
    def _find_desktop_files(self) -> Iterator[TargetPath]:
```
Please make the other "find_installs" type functions the same within this PR.